test/simple/test-repl-tab-complete.js

```diff
@@ -211,3 +211,36 @@ testMe.complete(' ', function(error, data) {
 testMe.complete('toSt', function(error, data) {
   assert.deepEqual(data, [['toString'], 'toSt']);
 });
+
+// To test custom completer function.
+var putIn2 = new ArrayStream();
+var testMe2 = repl.start({
+  prompt: '',
+  input : putIn2,
+  output: putIn2,
+  completer: function customCompleter(line, cb) {
+    var completions = 'aaa aa1 aa2 bbb bb1 bb2 bb3 ccc ddd eee'.split(' ');
```

Comment:
Instead of having all of these strings and calling `split()`, can you just start with arrays.